kayenta-influxdb/src/test/java/com/netflix/kayenta/influxdb/service/InfluxdbQueryBuilderTest.java

```diff
@@ -77,8 +77,7 @@ public void testBuild_withValidScope() {
     InfluxdbCanaryMetricSetQueryConfig queryConfig = queryConfig(measurement, fieldsList(), null);
     String query = queryBuilder.build(queryConfig, canaryScope);
     assertThat(
-        query,
-        is(
+        query.contains(
```

Comment:
`contains` is a bit less strict than `is`, no?  What about
```suggestion
        query).isEqualTo(
```
?

Reply:
Since, isEqualsTo() is not available as function, going to use equalsIgnoreCase().

Reply:
Update the code.